<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NMG-PRG-L-00027" failures="3" tests="19" name="org.example.TestRunner.Runners" time="231.847" errors="0" timestamp="2022-11-13T01:33:41 EET" skipped="0">
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="11.021"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="18.520"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="11.408"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="7.127"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="6.418">
    <failure message="Assertion 1st slider fail expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Assertion 1st slider fail expected [https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.example.StepDefs.D06_homeSlidersStepDef.userDirectToNokiaLumiaPage(D06_homeSlidersStepDef.java:26)
at ✽.user direct to Nokia Lumia page(file:///C:/Users/noura.zakarya/nopcommerce_project/src/main/resources/Ecommerce_Features/F06_HomeSliders.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="7.238"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="7.396"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="8.546"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="9.546"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="6.943"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="13.778"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="10.413"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="13.839"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="37.950">
    <failure message="Assertion 2nd slider fail expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Assertion 2nd slider fail expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.example.StepDefs.D06_homeSlidersStepDef.userDirectToIPhone(D06_homeSlidersStepDef.java:41)
at ✽.user direct to iPhone(file:///C:/Users/noura.zakarya/nopcommerce_project/src/main/resources/Ecommerce_Features/F06_HomeSliders.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="19.147"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="7.056"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="10.617"/>
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="16.568">
    <failure message="Assertion rss fail expected [https://demo.nopcommerce.com/new-online-store-is-open] but found [https://demo.nopcommerce.com/news/rss/1]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Assertion rss fail expected [https://demo.nopcommerce.com/new-online-store-is-open] but found [https://demo.nopcommerce.com/news/rss/1]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.example.StepDefs.D07_followUsStepDef.rssLinkOpensSuccessfully(D07_followUsStepDef.java:63)
at ✽.rss link "https://demo.nopcommerce.com/new-online-store-is-open"opens successfully(file:///C:/Users/noura.zakarya/nopcommerce_project/src/main/resources/Ecommerce_Features/F07_FollowsUs.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="org.example.TestRunner.Runners" name="runScenario" time="8.316"/>
  <system-out/>
</testsuite> <!-- org.example.TestRunner.Runners -->
